blueprints/grafana/panels/rabbitmq/rmq_static.libsonnet

```diff
@@ -0,0 +1,51 @@
+local utils = import '../../utils/policy_utils.libsonnet';
+local statPanel = import '../../utils/stat_panel.libsonnet';
+
+function(policyName, infraMeterName, datasource, extraFilters) {
+  local stringFilters = utils.dictToPrometheusFilter(extraFilters { policy_name: policyName }),
+
+  local ready = statPanel(
+    'Ready Messages',
+    datasource.name,
+    'sum(rabbitmq_message_current{%(filters)s, state="ready"})' % { filters: stringFilters },
+    stringFilters
+  ),
+
+  local unacknowledged = statPanel('Unacknowledged Messages',
+                                   datasource.name,
+                                   'sum(rabbitmq_message_current{%(filters)s, state="unacknowledged"})' % { filters: stringFilters },
+                                   stringFilters,
+                                   panelColor='red'),
+
+  local incoming = statPanel('Rate of Incoming Messages',
+                             datasource.name,
+                             'sum(rate(rabbitmq_message_published_total{%(filters)s}[$__rate_interval]))' % { filters: stringFilters },
```

Comment:
Need to use `__range` here since it's a non-time-series panel. And switch to instant query